models/log.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const logSchema = new mongoose.Schema(
+    {
+      exercise: String,
+  duration: {type: Number, min: 1, max: 60, required: true},
+  reps: {type: Number, min: 1, max: 100, required: true}
+ 
+    },
+    {
+      timestamps: true,
+    }
+  );
+
+
+
+  
+module.exports = mongoose.model('Log', logSchema);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7841248)
```suggestion
module.exports = mongoose.model("Log", logSchema);
```